src/frontend/context/MigreringContext.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import createUseContext from 'constate';
+
+import { useHttp } from '@navikt/familie-http';
+import { RessursStatus } from '@navikt/familie-typer';
+
+import { IInfotrygdSak, IInfotrygdsaker } from '../typer/infotrygd';
+import { Adressebeskyttelsegradering } from '../typer/person';
+
+const [MigreringProvider, useMigrering] = createUseContext(() => {
```

Comment:
```suggestion
const [InfotrygdProvider, useInfotrygd] = createUseContext(() => {
```

Gir det mening også her å snakke om infotrygd*. Videre når vi får en migreringsfunksjon så gir det også mening å legge det inn i en InfotrygdProvider tenker jeg :)

Reply:
Ikke sant - det var det jeg funderte litt på. Yep, infotrygd er nok litt mer "romslig" som begrep.